typespec/Applications.Dapr/secretStores.tsp

```diff
@@ -45,10 +45,15 @@ model DaprSecretStoreResource
   name: ResourceNameString;
 }
 
+// This is to prevent a secret store from referencing another secret store
+// This would be valid according to Dapr component specs, but may confuse users
+@withoutOmittedProperties("secretStoreComponentName")
+model NonRedundantDaprResourceProperties is DaprResourceProperties;
```

Comment:
Could there be a case where users can require authentication to a Secret Store?

Reply:
I think we're assuming the Dapr secret store has already been created.